packages/react/src/helpers/componentStackHelpers.ts

```diff
@@ -0,0 +1,29 @@
+import { BacktraceStackFrame } from '@backtrace/sdk-core/src/model/data/BacktraceStackTrace';
+
+// React 16 component stacks need to be parsed separately. React 17+ component stacks can be parsed like Error stacks
+export function isReact16ComponentStack(stack: string): boolean {
+    if (!stack) return false;
```

Comment:
```suggestion
    if (!stack) { 
        return false;
    }
```

can we continue using brackets? This is the pattern we have in the code.

Reply:
Fixed